streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import io.debezium.data.Envelope;
+import org.apache.camel.*;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.debezium.DebeziumConstants;
+import org.apache.camel.model.dataformat.JsonLibrary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+
+import static org.openmrs.analytics.PipelineConfig.getDebeziumConfig;
+
+/**
+ * Debezium change data capture / Listener
+ */
+public class DebeziumListener  extends RouteBuilder {
```

Comment:
Can we add some unit-tests for this class too as it has some non-trivial logic in it?

Reply:
Sure!

Reply:
I have added unit tests for all the  2 main classes